lib/ansible/playbook/attribute.py

```diff
@@ -155,6 +155,8 @@ def __init__(self, extend=False, prepend=False, **kwargs):
         self.prepend = prepend
 
     def __get__(self, obj, obj_type=None):
+        if obj is None:
+            return self
```

Comment:
@mkrizek is this OK with your planned changes?

Reply:
I suppose it does not break anything but trying to understand why is it needed for this PR in the comment above.

Reply:
i think the main issue is detecting 'unset' for this keyword, which having a default makes harder